deployment/docker-compose.yml

```diff
@@ -3,8 +3,10 @@ version: '3.4'
 # Common Django template for Geonode, Celery and Celerycam services below
 x-common-django:
   &default-common-django
+  image: gep-benin_django:latest
   build:
-    context: docker
+    context: ../
+    dockerfile: deployment/docker/Dockerfile
```

Comment:
I changed the context location because we need to include the `django_project` source code.